src/js/utils/time-ranges.js

```diff
@@ -25,8 +35,22 @@ export function createTimeRange(start, end){
     };
   }
   return {
-    length: 1,
-    start: function() { return start; },
-    end: function() { return end; }
+    length: ranges.length,
+    start: getRange.bind(null, 'start', 0, ranges),
+    end: getRange.bind(null, 'end', 1, ranges)
   };
 }
+
+function getRange(fnName, valueIndex, ranges, rangeIndex){
+  if (rangeIndex === undefined) {
```

Comment:
No args here actually throws an error with native timeranges. If we want them to be interchangeable we probably shouldn't let people do this.